nerdlets/groundskeeper-nerdlet/components/TableWrapper.js

```diff
@@ -19,82 +18,15 @@ export default class TableWrapper extends React.PureComponent {
     tableData: PropTypes.object.isRequired,
   };
 
-  renderHTMLTable() {
-    const {
-      tableData: { columns, data },
-    } = this.props;
-    const headers = columns.map(colName => (
-      <th key={`table-header-${colName}`}>{colName}</th>
-    ));
-    const rows = data.map((row, index) => {
-      return (
-        <tr key={`table-row-${index}`}>
-          {row.map((cell, cIndex) => (
-            <td key={`table-row-${index}-col-${cIndex}`}>{cell}</td>
-          ))}
-        </tr>
-      );
-    });
+  render() {
+    const { tableData } = this.props;
 
     return (
-      <table>
-        <thead>
-          <tr>{headers}</tr>
-        </thead>
-        <tbody>{rows}</tbody>
-      </table>
+      <BootstrapTable
```

Comment:
This is so simple now, seems like you could remove the tablewrapper component entirely.

Reply:
Good point. Looks like it wasn't even being used after these changes. Pulling it out now.